themes/gatsby-theme-blog/src/components/layout.js

```diff
@@ -35,6 +37,7 @@ const Title = props => {
   } else {
     return (
       <Styled.h3
+        as="p"
```

Comment:
NB: Would it make sense to use the same header and styles for both the index and post pages? It always seemed odd to me to have things shift vertically between the pages and could make sharing this header with the notes theme feel a bit nicer IMO – I'm also unsure how the `rootPath` will work with the notes and other themes composed with this one

Reply:
I think it's reasonable that people would want to be able to customize between the root header and slightly varied headers in different sections. So, more inconvenient but probably a problem we should discuss/tackle now.